packages/harmony/src/components/text-link/TextLink.tsx

```diff
@@ -17,26 +16,36 @@ export const TextLink = (props: TextLinkProps) => {
     onClick,
     textVariant,
     showUnderline,
+    applyHoverStylesToInnerSvg,
     ...other
   } = props
 
   const { color } = useTheme()
 
   const variantColors = {
     default: color.link.default,
+    subdued: color.link.subdued,
     visible: color.link.visible,
     inverted: color.static.white
   }
 
   const variantHoverColors = {
     default: color.primary.p300,
+    subdued: color.primary.p300,
     visible: color.link.visible,
     inverted: color.static.white
   }
 
   const hoverStyles = {
     textDecoration: 'underline',
-    color: variantHoverColors[variant]
+    color: variantHoverColors[variant],
+    ...(applyHoverStylesToInnerSvg
```

Comment:
interesting... dont love it but it makes sense!

Reply:
all ears for a better way, but it was naive of me to just roll for it as a global change on the old `ExternalLink` code